### ISSUE TITLE
[eBUG]ArrayIndexOutOfBoundsException when Skilltester saved temporary player

### ISSUE THREAD

**Issue by wsbrenk** [akasolace/HO#446]
![grafik](https://user-images.githubusercontent.com/11821502/79106954-c6e79c00-7d73-11ea-8317-9d1044221868.png)

Push the button "Add player" in Skilltester.
Then adjust Player overview to show new player in the list
then this exception occures

java.lang.ArrayIndexOutOfBoundsException: 30
	at core.gui.comp.table.TableSorter.getValueAt(TableSorter.java:182)
	at core.gui.model.ReduzedTableModel.getValueAt(ReduzedTableModel.java:56)
	at javax.swing.JTable.getValueAt(JTable.java:2720)
	at javax.swing.JTable.prepareRenderer(JTable.java:5712)
	at javax.swing.plaf.synth.SynthTableUI.paintCell(SynthTableUI.java:683)
	at javax.swing.plaf.synth.SynthTableUI.paintCells(SynthTableUI.java:580)
	at javax.swing.plaf.synth.SynthTableUI.paint(SynthTableUI.java:364)
	at javax.swing.plaf.synth.SynthTableUI.update(SynthTableUI.java:275)
	at javax.swing.JComponent.paintComponent(JComponent.java:780)
	at javax.swing.JComponent.paint(JComponent.java:1056)
	at javax.swing.JComponent.paintChildren(JComponent.java:889)
	at javax.swing.JComponent.paint(JComponent.java:1065)
	at javax.swing.JViewport.paint(JViewport.java:728)
	at javax.swing.JComponent.paintChildren(JComponent.java:889)
	at javax.swing.JComponent.paint(JComponent.java:1065)
	at javax.swing.JComponent.paintChildren(JComponent.java:889)
	at javax.swing.JSplitPane.paintChildren(JSplitPane.java:1047)
	at javax.swing.JComponent.paint(JComponent.java:1065)

**Comment by tychobrailleur** [comment]
Is that issue a resurgence of #401 ? It seems to be the same exception.

I am not sure how the fix relates to it, though: I get an NPE when trying to create a Skilltester player which makes sense in the light of the fix #447 :
```
 [Error]   RefreshManager: java.lang.NullPointerException
 [Error]   RefreshManager: core.model.player.Player.getAgeWithDaysAsString(Player.java:609)
 [Error]   RefreshManager: core.model.player.Player.getAgeWithDaysAsString(Player.java:604)
 [Error]   RefreshManager: core.model.player.Player.getAlterWithAgeDaysAsString(Player.java:600)
 [Error]   RefreshManager: core.gui.model.UserColumnFactory$26.getTableEntry(UserColumnFactory.java:738)
 [Error]   RefreshManager: core.gui.model.PlayerOverviewModel.initData(PlayerOverviewModel.java:163)
 [Error]   RefreshManager: core.gui.model.PlayerOverviewModel.setValues(PlayerOverviewModel.java:89)
 [Error]   RefreshManager: module.playerOverview.PlayerOverviewTable.initModel(PlayerOverviewTable.java:177)
 [Error]   RefreshManager: module.playerOverview.PlayerOverviewTable.reInit(PlayerOverviewTable.java:88)
 [Error]   RefreshManager: core.gui.RefreshManager.doReInit(RefreshManager.java:48)
 [Error]   RefreshManager: module.playerOverview.SpielerTrainingsSimulatorPanel.actionPerformed(SpielerTrainingsSimulatorPanel.java:216)

(. . .)
 [Error]   RefreshManager: java.lang.NullPointerException
```

Does that issue then cause the AOOBE for you?  Let me know how to reproduce, happy to look into it as I recently worked on `TableSorter`. ;-)

**Comment by wsbrenk** [comment]
> Does that issue then cause the AOOBE for you? 

Yes it did. it was catched by RefreshManager.doReInit and in consequence the given exception arises.

**Comment by wsbrenk** [comment]
> I recently worked on TableSorter. ;-)

by the way. the order after the first sort click is since some weeks an ascending one. I was used to the opposite. High skills at the top.
Now i need 2 clicks to get what i want;-)

**Comment by tychobrailleur** [comment]
> > I recently worked on TableSorter. ;-)
> 
> by the way. the order after the first sort click is since some weeks an ascending one. I was used to the opposite. High skills at the top.
> Now i need 2 clicks to get what i want;-)

Ah sorry about that...  Ascending sounds like the logical intuitive behaviour, but we could also make this configurable, what do you think?

**Comment by akasolace** [comment]
closed by #447

**Comment by wsbrenk** [comment]
> we could also make this configurable, what do you think?

comfortable++
